understanding/contrast-minimum.html

```diff
@@ -135,236 +131,119 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
-                     different for this population. However, color deficiencies are so diverse that prescribing
-                     effective general use color pairs (for contrast) based on quantitative data is not
-                     feasible. Requiring good luminance contrast accommodates this by requiring contrast
-                     that is independent of color perception. Fortunately, most of the luminance contribution
-                     is from the mid and long wave receptors which largely overlap in their spectral responses.
-                     The result is that effective luminance contrast can generally be computed without
-                     regard to specific color deficiency, except for the use of predominantly long wavelength
-                     colors against darker colors (generally appearing black) for those who have protanopia.
-                     (We provide an advisory technique on avoiding red on black for that reason). For more
-                     information see [[ARDITI-KNOBLAUCH-1994]] 
-                     [[ARDITI-KNOBLAUCH-1996]] 
-                     [[ARDITI]].
+                  <p>色相は、色覚異常 (先天性及び後天性の両方) をもつ利用者によって、通常の目の見える利用者とは異なる色及び相対的な輝度コントラストを結果としてもたらすように、異なって知覚される。このため、有効なコントラスト及び読みやすさは、人々によって異なる。しかし、色覚異常は非常に多様であるため、定量的なデータに基づく (コントラストのための) 有効で一般的な色の組み合わせを規定することは、実現不可能である。十分な輝度コントラストを要求することは、色の知覚から独立したコントラストを要求することによって、これに対応している。幸いにも、輝度の寄与のほとんどは、分光応答で大部分が重なる中波長及び長波長の受容体からのものである。その結果、有効な輝度コントラストが一般的に、特定の色覚異常に関係なく計算できる。ただし、赤色を知覚しにくい 1 型 2 色覚の人の場合、暗い色 (一般に黒に見える) に対して主に長波長の色を用いることは除く (この理由のために、WCAG ワーキンググループは、黒の背景色に赤の前景色を使うことを避ける参考達成方法を提供している)。詳細については、[ARDITI-KNOBLAUCH] [ARDITI-KNOBLAUCH-1996] [ARDITI] を参照のこと。
                      
                   </p>
                   
                   
                   <div class="note">
-                     <p class="note-title marker">Note</p>
+                     <p class="note-title marker">注記</p>
                      <div>
                         
                         
-                        <p>Some people with cognitive disabilities require color combinations or hues that have
-                           low contrast, and therefore we allow and encourage authors to provide mechanisms to
-                           adjust the foreground and background colors of the content. Some of the combinations
-                           that could be chosen may have contrast levels that will be lower than those those
-                           specified here. This is not a violation of this Success Criterion, provided
-                           there is a mechanism that will return to the required values set out here.
+                        <p>認知の障害のある人の中には、コントラストの低い色の組み合わせ又は色相を必要とすることがある。そのため、コンテンツ制作者にコンテンツの前景色と背景色を調節するメカニズムを提供することを認め、それを推奨する。選択可能な組み合わせの中には、ここで指定されているより低いレベルのコントラストがあってもよい。ここで要求されている指定値に戻すことができるメカニズムが提供されていれば、この達成基準に反していることにはならない。
```

Comment:
冒頭の文は「人の中には……することがある」となっていますが、「人の中には……する人もいる」「人は……することもある」のいずれかかと思います。

```suggestion
                        <p>認知の障害のある人は、コントラストの低い色の組み合わせ又は色相を必要とすることもある。そのため、コンテンツ制作者にコンテンツの前景色と背景色を調節するメカニズムを提供することを認め、それを推奨する。選択可能な組み合わせの中には、ここで指定されているより低いレベルのコントラストがあってもよい。ここで要求されている指定値に戻すことができるメカニズムが提供されていれば、この達成基準に反していることにはならない。
```